erpnext/healthcare/doctype/vital_signs/vital_signs.json

```diff
@@ -92,16 +90,16 @@
    "fieldtype": "Column Break"
   },
   {
-   "default": "Today",
-   "fieldname": "signs_date",
-   "fieldtype": "Date",
-   "label": "Date",
-   "reqd": 1
-  },
-  {
-   "fieldname": "signs_time",
-   "fieldtype": "Time",
-   "label": "Time",
+    "default": "Today",
+    "fieldname": "signs_date",
+    "fieldtype": "Date",
+    "label": "Date",
+    "reqd": 1
+   },
+   {
+    "fieldname": "signs_time",
+    "fieldtype": "Time",
+    "label": "Time",
    "reqd": 1
```

Comment:
@mujeerhashmi Can you please revert this?

Reply:
@ChillarAnand Done